dashboard/modules/data/data_head.py

```diff
@@ -0,0 +1,95 @@
+import json
+import os
+from enum import Enum
+import aiohttp
+from aiohttp.web import Request, Response
+import ray.dashboard.optional_utils as optional_utils
+import ray.dashboard.utils as dashboard_utils
+from ray.data._internal.stats import _get_or_create_stats_actor
+from ray.dashboard.modules.metrics.metrics_head import (
+    PROMETHEUS_HOST_ENV_VAR,
+    DEFAULT_PROMETHEUS_HOST,
+    PrometheusQueryError,
+)
+from urllib.parse import quote
+import ray
+
+
+MAX_TIME_WINDOW = "1h"
+SAMPLE_RATE = "1s"
+
+
+class PrometheusQuery(Enum):
+    VALUE = ("value", "sum({}) by (dataset)")
+    MAX = (
+        "max",
+        "max_over_time(sum({}) by (dataset)[" + f"{MAX_TIME_WINDOW}:{SAMPLE_RATE}])",
+    )
```

Comment:
Is there a simpler way to get the max of a metric?

Reply:
I think something like `[1h:1s]` in this case means that we're querying the last 1 hour at 1 second intervals.